### PR TITLE
Add Turkish Translations to Messages

### PR DESCRIPTION
**Pull request by serkanz**

### CODE REVIEW

**Comment by serkanz** [comment]
#1051

File: src/OpenIddict.Abstractions/Resources/xlf/OpenIddictResources.tr.xlf [hunk 1]
```diff
@@ -1,600 +1,600 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
   <file datatype="xml" source-language="en" target-language="tr" original="../OpenIddictResources.resx">
     <body>
       <trans-unit id="ID3000">
         <source>The security token is missing.</source>
-        <target state="new">The security token is missing.</target>
+        <target state="new">Güvenlik belirteci eksik.</target>
```
Comment (kevinchalet) [inline src/OpenIddict.Abstractions/Resources/xlf/OpenIddictResources.tr.xlf]: Can you please change the `state` of all the translated strings to `translated`?

**Review by kevinchalet — APPROVED** [review]
Thanks for your PR!

File: src/OpenIddict.Abstractions/Resources/xlf/OpenIddictResources.tr.xlf (hunk 1 above)
Comment (serkanz) [reply src/OpenIddict.Abstractions/Resources/xlf/OpenIddictResources.tr.xlf]: Ok, I committed states.

**Comment by kevinchalet** [comment]
Thanks for your contribution!

**Comment by candidodmv** [comment]
Hello @kevinchalet , how could I procceed to add support for pt-BR(portuguese brazilian) ?

**Comment by kevinchalet** [comment]
Hey @candidodmv,

As explained in https://kevinchalet.com/2020/11/17/introducing-openiddict-3-0-s-first-release-candidate-version/, localization support sadly had to be removed. That said, I plan to re-use the localized strings provided by the community in the online documentation. If you're interested in helping, you can send PRs to document the errors returned by OpenIddict: https://github.com/openiddict/openiddict-documentation/issues/30.

Cheers.

**Comment by candidodmv** [comment]
Sorry, but I din't find the file base(neither its directory) in the project structure at your mentioned link:
[openiddict/openiddict-documentation#30.](https://github.com/openiddict/openiddict-documentation/issues/30)

could you please, show where exactly the resource file base reside for me use in the translation?

**Pull request merged**